src/status_im/common/enter_seed_phrase/view.cljs

```diff
@@ -147,22 +151,24 @@
          [rn/view {:style style/keyboard-container}
           [quo/predictive-keyboard
            {:type     suggestions-state
-            :blur?    true
+            :blur?    (not keypair?)
             :text     suggestions-text
             :words    (keyboard-suggestions last-word)
             :on-press pick-suggested-word}]])])
     (finally
      (.remove keyboard-show-listener)
      (.remove keyboard-hide-listener))))
 
-(defn enter-seed-phrase
+(defn view
   []
   (let [{navigation-bar-top :top} (safe-area/get-insets)]
-    [rn/view {:style style/full-layout}
-     [rn/keyboard-avoiding-view {:style style/page-container}
-      [quo/page-nav
-       {:margin-top navigation-bar-top
-        :background :blur
-        :icon-name  :i/arrow-left
-        :on-press   #(rf/dispatch [:navigate-back])}]
-      [screen]]]))
+    (fn []
+      (let [{:keys [keypair?]} (rf/sub [:get-screen-params])]
```

Comment:
keypair? is a big ambiguous imo. What does this imply/ is it used for?

Reply:
Means are we recovering a keypair ?

Changed it to `recovering-keypair?`